profiles/base.yaml

```diff
@@ -98,7 +98,7 @@ manipulate_only_to_train: false
 # See combat-trainer documentation above for some examples.
 training_abilities:
   Hunt: 80
-# how high in mind-states to train training_abilities before skipping them
+# how high in mind-states to train training_abilities before skipping them 
```

Comment:
❓ What's with the whitespace changes in this file?

Reply:
I do not know. I will try to fix. Since I was in dev mode working on enchanting, many changes went in. I have tried to make sure the files were updated to match existing and then my changes.

Reply:
Think a commit I did just before this one trimmed all the trailing whitespaces.
There is a menu option in notepad++ that lets you do that very easily.

Reply:
I believe I did it.